python/matcher.go

```diff
@@ -19,40 +19,54 @@ type Matcher struct{}
 // Name implements driver.Matcher.
 func (*Matcher) Name() string { return "python" }
 
-// Filter implements driver.Matcher.
-func (*Matcher) Filter(record *claircore.IndexRecord) bool {
-	return record.Package.NormalizedVersion.Kind == "pep440"
+func (*Matcher) Filter(r *claircore.IndexRecord) bool {
```

Comment:
I think the `pep440` check still makes sense for this, but I'm not 100% sure. I'll defer this to someone else

Reply:
Yeah, I think it still does, even if only in a belt-and-suspenders way.